.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": [
+    "eslint:recommended"
+  ],
+  "plugins": [
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "rules": {
+    "no-unused-vars": [
+      "warn",
+      {
+        "argsIgnorePattern": "^_",
+        "varsIgnorePattern": "^_",
+        "caughtErrorsIgnorePattern": "^_"
+      }
+    ],
+    "no-cond-assign": "warn",
```

Comment:
I suggest we change this to "off" given it's used fairly extensively. Warnings are just annoying.